index.html

```diff
@@ -127,77 +127,142 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
+          can be used to remove the H.264/AVC codec from the Offer, improving the
+          chances that a codec supporting temporal scalability is negotiated.
         </p>
         <p>
           When {{RTCRtpTransceiverInit/sendEncodings}} is used to
           request the sending of multiple simulcast streams using
-          {{RTCPeerConnection/addTransceiver()}}, it is not possible to
-          configure the sending of "S" scalability modes. The browser
-          may only be configured to send simulcast encodings with
-          multiple SSRCs and RIDs, or alternatively, to send all
-          simulcast encodings on a single RTP stream. The
-          <a>user agent</a>  MUST return {{OperationError}} in
-          {{RTCRtpSender/setParameters()}} or
-          {{RTCPeerConnection/addTransceiver()}} in response to
-          an attempt to simultaneously utilize both simulcast transport
-          techniques.
+          {{RTCPeerConnection/addTransceiver()}}, an "S mode" cannot
+          be requested. The browser may only be configured to send
+          simulcast encodings with multiple SSRCs and RIDs, or
+          alternatively, to send all simulcast encodings on a single
+          RTP stream. Simultaneously using both simulcast transport
+          techniques is not permitted. 
         </p>
+      </section>
+      <section id="setparameters">
+       <h3>{{RTCRtpSender/setParameters()}}</h3>
+         <p>
+           [[!WEBRTC]] Section 5.2 describes validation of <var>parameters</var>
+           within {{RTCRtpSender/setParameters()}}. Add the following to the 
+           conditions (step 4 within step 6) for an {{InvalidModificationError}}:
+           <ol>
+             <li>
+               <var>encodings</var> contains an encoding whose
+               {{RTCRtpEncodingParameters/scalabilityMode}} value is
+               not supported by any codec in <var>codecs</var>.
+             </li>
+             <li>
+               <var>N</var> is greater than 1, and <var>encodings</var>
+               contains an encoding whose
+               {{RTCRtpEncodingParameters/scalabilityMode}} value
+               represents an "S mode".
+             </li>
+           </ol>
+         </p>
+      </section>
+      <section id="getparameters">
+       <h3>{{RTCRtpSender/getParameters()}}</h3>
+         <p>
+           To determine whether {{RTCRtpEncodingParameters/scalabilityMode}}
+           values requested via {{RTCPeerConnection/addTransceiver()}} or
+           {{RTCRtpSender/setParameters()}} have been applied,
+           an application can call the {{RTCRtpSender.getParameters()}} method
+           after negotiation has completed and the sending codec has been
+           determined. If the configuration is not satisfactory,
+           {{RTCRtpSender/setParameters()}} can be used to change it.
+         </p>
+         <p>
+           {{RTCRtpSender/getParameters()}} returns the currently
```

Comment:
This behaviour is after the negotiation is completed, right? If so, I would add it explicitly and reorder it so it appears after all the "before negotiation completes" description

Reply:
Clarified.